chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertificationResult.java

```diff
@@ -220,7 +220,10 @@
     private List<MacraMeasure> g2MacraMeasures = new ArrayList<MacraMeasure>();
 
     /**
-     * NEED TO GET THIS TEXT
+     * ONC has established the Standards Version Advancement Process (SVAP) to enable health IT developers’
+     * ability to incorporate newer versions of Secretary-adopted standards and implementation specifications,
+     * as part of the "Real World Testing" Condition and Maintenance of Certification requirement (§170.405)
```

Comment:
Does this `§` character show up correctly in the XSD?

Reply:
Yep - 
![image](https://user-images.githubusercontent.com/24817202/97442103-db4e4900-18ff-11eb-9036-9e8c5c0f1cae.png)